### DISCUSSION TITLE
Disable Log Output

### DISCUSSION THREAD

**Discussion by dennisheerwb** [capacitor-community/bluetooth-le#625] [Q&A] [answered]
`BluetoothLe` is repeatedly logging to the console for events such as `startLEScan`, `stopLEScan`, `addListener`, and `removeListener`.

These log messages pollute the log output and regularly result in Safari dev tools crashing. I was not able to find any information on how to configure the log verbosity or turn logging off.

Is there a way to disable logging or reduce its verbosity?

<img width="530" alt="image" src="https://github.com/capacitor-community/bluetooth-le/assets/108330177/e47ea9d2-236b-4718-93b0-465c2e58ddc7">

**Comment by pwespi** [comment, marked as answer]
There is the `loggingBehavior` option you can set in the Capacitor config file, see https://capacitorjs.com/docs/config.
